src/include/cart/types.h

```diff
@@ -416,6 +416,13 @@ typedef enum {
  */
 #define CRT_RPC_FEAT_NO_TIMEOUT		(1U << 2)
 
+/**
+ * If RPC ends up being queued due to exceeding in-flight rpc limit, queue
+ * at the front of the queue. If not set, queues at the end
+ */
+#define CRT_RPC_FEAT_QUEUE_FRONT	(1U << 3)
```

Comment:
Do we want this to be a public flag or a private one?

Reply:
I wanted to give end user ability to select, in case they have their own management or high priority rpcs that should go in front on queue (based on their particular use-case)